crdppf/util/pdf_functions.py

```diff
@@ -324,9 +324,11 @@ def get_feature_info(request, translations):
     for debbuging the query use str(query) in the console/browser window
     to visualize geom.wkt use session.scalar(geom.wkt)
     """
-
-    SRS = 21781
-
+    try: 
```

Comment:
You do not need this.... If srid is not defined, then this function will throw an error... `def function(srid)` assumes that an srid parameter is passed to the function when this one is called. Calling the function without this parameter will result in an error.